src/Mail/exceptions.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * This file is part of the Nette Framework (http://nette.org)
+ * Copyright (c) 2004 David Grudl (http://davidgrudl.com)
+ */
+
+namespace Nette\Mail;
+
+use Nette;
+
+
+/**
+ * Send failed exception.
+ *
+ * @author Jan Dvořák
+ */
+class SendFailedException extends Nette\InvalidStateException
```

Comment:
To jo, ale stačilo by přidat obecnou SendFailedException a SmtpException od ní podědit.